packages/flutter/test/material/button_theme_test.dart

```diff
@@ -402,4 +405,86 @@ void main() {
   },
     semanticsEnabled: true,
   );
+
+  testWidgets('default button theme accent color for RaisedButton', (WidgetTester tester) async { 
+    const Color defaultColor = Color(0xff2196f3);
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Scaffold(
+          body: Center(
+            child: RaisedButton(
+              child: const Text('RaisedButton'),
+              onPressed: () {},
+              textTheme: ButtonTextTheme.accent,
+            ),
+          ),
+        ),
+      ),
+    );
+
+    Color textColor() {
+      return tester.renderObject<RenderParagraph>(find.text('RaisedButton')).text.style.color;
+    }
+
+    // Default, not disabled.
```

Comment:
With improved variable names, this comment is redundant
```suggestion
```

Reply:
removed it